content/posts/why-meson/index.md

```diff
@@ -0,0 +1,174 @@
++++
+"author" = "ret2libc"
+"title" = "Why meson"
+"date" = "2020-12-11"
+"summary" = "Why we chose meson"
```

Comment:
make it longer. "Why we chose meso as the building system for Rizin?" or "Why did we left XYZ and moved to Meson as our building system?" etc

Reply:
I think it's better to clarify even further: "Why we chose Meson as the default building system for Rizin?" or "Why we chose Meson over ACR+Makefiles as the default building system for Rizin?"